charts/testkube-enterprise/values.yaml

```diff
@@ -238,6 +238,27 @@ testkube-worker-service:
   image:
     repository: testkubeenterprise/testkube-enterprise-worker-service
     tag: 1.9.0-rc-e5167f6a
+
+testkube-logs-service:
```

Comment:
also please duplicate these values to a `local-values.yaml` file that is used for simplified Enterprise installation.

Reply:
done :)